src/content/recipes/tailwindcss.md

```diff
@@ -26,7 +26,25 @@ In this post, we will:
 
 ![Finished example of Tailwind CSS in Storybook with a theme switcher](https://user-images.githubusercontent.com/18172605/208201389-1f448dbb-978c-442e-9d6b-7bf3fea63e64.gif)
 
-## Build Tailwind next to Storybook
+## Auto-config
+
+As of version 1.1, `@storybook/addon-styling` offers a codemod for to automatically configure your storybook with Tailwind.
+
+To try it out, run the following commands:
+
+```shell
+# Install the addon
+yarn add -D @storybook/addon-styling
+
+# Run the postinstall script from the root of your project
+node node_modules/@storybook/addon-styling/bin/postinstall.js
+```
+
+If the codemod didn't work, please let us know in [this GitHub issue](https://github.com/storybookjs/addon-styling/issues/49#issue-1746365130) so we can continue to make the codemod even better. In the meantime, the instructions below will get you up and running in no time.
+
+## Manual
```

Comment:
Per [this earlier comment](https://github.com/storybookjs/frontpage/pull/564/files#r1229837628), the install instructions either need to move out of and before the Auto-config & Manual sections, or they need to be included in each (for now).